### ISSUE TITLE
Questions about data preprocessing

### ISSUE THREAD

**Issue by YimengQiao** [TencentAILabHealthcare/scTranslator#9]
Dear authors, thanks for your great work! I believe this is a leading job in this field.

I would like to ask what data preprocessing methods you adopted for the RNA and protein data in the pretraining process. Besides the min-max normalization you mentioned in the paper, have you ever performed or tried other typical single-cell data preprocessing steps before that, such as normalize_total, log1p, HVG selection, etc.? 

Looking forward to your reply!

**Comment by YimengQiao** [comment]
Besides, I am also very curious about how much the bulk-level training data contributed to the single-cell level protein prediction task. If we just use the single-cell level data for pre-training, would the single-cell level prediction performance decrease significantly? In turn, if we use the model pre-trained with both bulk-level and single-cell level data for the bulk-level prediction task, would it be better than the model pre-trained with bulk-level data only? (just out of curiosity)
